orbit/pkg/packaging/linux_shared.go

```diff
@@ -192,7 +193,7 @@ func buildNFPM(opt Options, pkger nfpm.Packager) (string, error) {
 		contents = append(contents, (&files.Content{
 			Destination: emptyFolder,
 			Type:        "dir",
-		}).WithFileInfoDefaults())
+		}).WithFileInfoDefaults(0o0, time.Now()))
```

Comment:
`WithFileInfoDefaults` now expects a `umask` and an `mtime` to use if the file info doesn't include permissions or timestamp data.  Since we weren't using this before, I'm providing a umask that's a no-op and a fallback mtime of "now" since that's [what it did before](https://github.com/goreleaser/nfpm/blame/c3142513c990e867c93c1acad256c9960a059fa1/files/files.go#L132).  Since these are new folders I don't think this should cause any problems (the umask is intended to provide sane permissions in cases where a user may have changed perms on a file locally before adding it to a package).